letsencrypt/data/run.sh

```diff
@@ -124,6 +124,6 @@ else
 fi
 
 # Get the last modified cert directory and copy the cert and private key to store
-CERT_DIR=$(find $CERT_DIR/live/* -type d -maxdepth 1 | awk '{ "stat -c '%Y' "$0 | getline ts; if (ts >= ots) printf "%d\t%s\n", ts, $0; ots=ts; }' | sort -k1 | tail -1 | awk '{ print $2 }')
+CERT_DIR=$(find $CERT_DIR/live/* -type d -maxdepth 1 | awk '{ "stat -c '%Y' "$0 | getline ts; if (ts >= ots) printf "%d\t%s\n", ts, $0; ots=ts; }' | sort -nk1 | tail -1 | awk '{ print $2 }')
```

Comment:
It's not the prettiest (and personally it's a bit overkill, just to be 100% on ShellCheck) but it works. 😄

`find $CERT_DIR/live/* -type d -maxdepth 1 `
Find all directories in /data/letsencrypt/live/

`awk '{ "stat -c '%Y' "$0 | getline ts; if (ts >= ots) printf "%d\t%s\n", ts, $0; ots=ts; }' `
Get the second since Epoch when the directory was modified and print it in the format "seconds_since_epoch(tab)directory_path"

`sort -nk1 | tail -1 | awk '{ print $2 }'`
Sort them numerically, by the first field (the Epoch), show only the last line, and show only the second field (the full path)

While I found that it worked in my own environment, it may be a good idea to tinker with it and test that there aren't any obvious edge cases that I didn't come across.